src/cogent3/core/tree.py

```diff
@@ -46,11 +46,11 @@
     overload,
 )
 
+from scipy.stats import pearsonr
```

Comment:
If you put this import inside the function, which I note is also being deprecated, then we can just delete the entire function instead of having to find this import and delete it too.